### ISSUE TITLE
Multiclass training 

### ISSUE THREAD

**Issue by abhigoku10** [traveller59/second.pytorch#393]
@dashidhy @traveller59  Hi thanks for opensourcing the work , I am using your multiclass traninning approach for my other architecture on Kitti dataset and i am having the following issue , can you share some pointers 
1. the set_guidedanchors() is giving out 0 values after a certain iteration 
2. the prediction score of my classification loss i not high so i am not able to get the any proper predictions 
3. Since its multiclass should not we use softmax then sigmoid for losses

THanks in advance

**Comment by CNWindson** [comment]
I want to add Truck detection to the code and I failed. Can you give me some advice?
